erpnext/manufacturing/doctype/job_card/job_card.py

```diff
@@ -67,14 +71,14 @@ def get_overlap_for(self, args, check_next_available_slot=False):
 			extra_cond = " or (%(from_time)s <= jctl.from_time and %(to_time)s <= jctl.to_time)"
 
 		existing = frappe.db.sql("""select jc.name as name, jctl.to_time from
```

Comment:
```suggestion
		existing = frappe.db.sql(f"""select jc.name as name, jctl.to_time from
```

It's quite difficult to follow `.format()` when replacement values are not in order. We can use f-strings now.